diff --git a/puppet-manifests/src/modules/platform/manifests/params.pp b/puppet-manifests/src/modules/platform/manifests/params.pp
index 587757b..4a9664b 100644
--- a/puppet-manifests/src/modules/platform/manifests/params.pp
+++ b/puppet-manifests/src/modules/platform/manifests/params.pp
@@ -37,8 +37,9 @@ class platform::params (
   $protected_group_id = '345'
 
   # PUPPET 4 treats custom facts as strings. We convert to int by adding zero.
-  $phys_core_count = 0 + $::physical_core_count
-  $plat_res_mem = 0 + $::platform_res_mem
+  $phys_core_count = Integer($::physical_core_count)
+  $plat_res_mem = Integer($::platform_res_mem)
+
 
   # Engineering parameters common to openstack services:
 
