From 35907952c646b971ba5b14002db2aac8d2324f21 Mon Sep 17 00:00:00 2001
From: babak sarashki <babak.sarashki@windriver.com>
Date: Sun, 3 Nov 2019 14:30:27 -0800
Subject: [PATCH 02/20] Correct log levels in ppolicy overlay

From STX 1901 openldap-ppolicy-loglevels.patch
---
 servers/slapd/overlays/ppolicy.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/servers/slapd/overlays/ppolicy.c b/servers/slapd/overlays/ppolicy.c
index 7d26b73..3f41e27 100644
--- a/servers/slapd/overlays/ppolicy.c
+++ b/servers/slapd/overlays/ppolicy.c
@@ -685,7 +685,7 @@ check_password_quality( struct berval *cred, PassPolicy *pp, LDAPPasswordPolicyE
 		if ((mod = lt_dlopen( pp->pwdCheckModule )) == NULL) {
 			err = lt_dlerror();
 
-			Debug(LDAP_DEBUG_ANY,
+			Log3(LDAP_DEBUG_ANY, LDAP_LEVEL_WARNING,
 			"check_password_quality: lt_dlopen failed: (%s) %s.\n",
 				pp->pwdCheckModule, err, 0 );
 			ok = LDAP_OTHER; /* internal error */
@@ -700,7 +700,7 @@ check_password_quality( struct berval *cred, PassPolicy *pp, LDAPPasswordPolicyE
 			if ((prog = lt_dlsym( mod, "check_password" )) == NULL) {
 				err = lt_dlerror();
 			    
-				Debug(LDAP_DEBUG_ANY,
+				Log3(LDAP_DEBUG_ANY, LDAP_LEVEL_WARNING,
 					"check_password_quality: lt_dlsym failed: (%s) %s.\n",
 					pp->pwdCheckModule, err, 0 );
 				ok = LDAP_OTHER;
@@ -709,7 +709,7 @@ check_password_quality( struct berval *cred, PassPolicy *pp, LDAPPasswordPolicyE
 				ok = prog( ptr, txt, e );
 				ldap_pvt_thread_mutex_unlock( &chk_syntax_mutex );
 				if (ok != LDAP_SUCCESS) {
-					Debug(LDAP_DEBUG_ANY,
+					Log3(LDAP_DEBUG_ANY, LDAP_LEVEL_WARNING,
 						"check_password_quality: module error: (%s) %s.[%d]\n",
 						pp->pwdCheckModule, *txt ? *txt : "", ok );
 				}
-- 
2.17.1

