diff -ruN a/sysinv/sysinv/sysinv/sysinv/puppet/interface.py b/sysinv/sysinv/sysinv/sysinv/puppet/interface.py
--- a/sysinv/sysinv/sysinv/sysinv/puppet/interface.py	2020-05-19 09:09:32.685727476 +0000
+++ b/sysinv/sysinv/sysinv/sysinv/puppet/interface.py	2020-05-19 09:10:34.225727195 +0000
@@ -938,8 +938,10 @@
             # others share the same pci-sriov sysfs enabling mechanism
             sriovfs_path = ("/sys/class/net/%s/device/sriov_numvfs" %
                             get_interface_port_name(context, iface))
-            options['pre_up'] = "echo 0 > %s; echo %s > %s" % (
-                sriovfs_path, iface['sriov_numvfs'], sriovfs_path)
+            sriovpf_up_cmd = ("ifconfig %s up" %
+                            get_interface_port_name(context, iface))
+            options['pre_up'] = "echo 0 > %s; echo %s > %s; %s" % (
+                sriovfs_path, iface['sriov_numvfs'], sriovfs_path, sriovpf_up_cmd)
     elif interface_class == constants.INTERFACE_CLASS_PCI_PASSTHROUGH:
         sriovfs_path = ("/sys/class/net/%s/device/sriov_numvfs" %
                         get_interface_port_name(context, iface))
