diff --git a/src/dnsmasq.c b/src/dnsmasq.c
index e1d3bbd..99e5437 100644
--- a/src/dnsmasq.c
+++ b/src/dnsmasq.c
@@ -358,7 +358,8 @@ int main (int argc, char **argv)
     }
 
 #ifdef HAVE_INOTIFY
-  if (daemon->port != 0 || daemon->dhcp || daemon->doing_dhcp6)
+  if ((daemon->port != 0 || daemon->dhcp || daemon->doing_dhcp6)
+      && (!option_bool(OPT_NO_RESOLV) || daemon->dynamic_dirs))
     inotify_dnsmasq_init();
   else
     daemon->inotifyfd = -1;
