diff -ru a/manifests/params.pp b/manifests/params.pp
--- a/manifests/params.pp	2020-03-05 14:17:45.430014390 +0800
+++ b/manifests/params.pp	2020-03-05 14:17:54.642014641 +0800
@@ -34,7 +34,11 @@
       $apache_group                = 'www-data'
       $wsgi_user                   = 'horizon'
       $wsgi_group                  = 'horizon'
-      $memcache_package            = 'python-memcache'
+      if $::operatingsystem == 'poky-stx' {
+          $memcache_package            = 'python-memcached'
+      } else {
+          $memcache_package            = 'python-memcache'
+      }
       case $::os_package_type {
         'debian': {
             $package_name          = 'openstack-dashboard-apache'
