--- /dev/null	2019-10-23 20:19:32.602763995 -0700
+++ b/puppet-staging.gemspec	2019-10-28 08:57:41.431851091 -0700
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name        = 'puppet-staging'
+  s.version     = '1.0.4'
+  s.date        = '2015-01-29'
+  s.summary     = "Puppet staging module"
+  s.description = s.summary
+  s.authors     = ["Puppet nanliu"]
+  s.email       = ''
+  s.files       = %w(LICENSE README.md Rakefile) + Dir.glob('{lib,spec}/**/*')
+  s.homepage    = 'https://github.com/nanliu/puppet-staging'
+  s.license     = 'Apache 2.0'
+ end
