diff -Nrupd a/lib/facter/operatingsystem/linux.rb b/lib/facter/operatingsystem/linux.rb
--- a/lib/facter/operatingsystem/linux.rb	2019-11-03 14:24:10.987872906 -0800
+++ b/lib/facter/operatingsystem/linux.rb	2019-11-03 14:25:31.222743152 -0800
@@ -13,6 +13,8 @@ module Facter
             @operatingsystem ||= "Ubuntu"
           elsif lsbdistid == "LinuxMint"
             @operatingsystem ||= "LinuxMint"
+          elsif lsbdistid == "poky"
+            @operatingsystem ||= "CentOS"
           else
             @operatingsystem ||= get_operatingsystem_with_release_files
           end
