diff -ru a/puppet-manifests/src/modules/platform/manifests/network.pp b/puppet-manifests/src/modules/platform/manifests/network.pp
--- a/puppet-manifests/src/modules/platform/manifests/network.pp	2020-04-08 19:10:04.478548420 +0800
+++ b/puppet-manifests/src/modules/platform/manifests/network.pp	2020-04-08 19:10:22.674548916 +0800
@@ -208,8 +208,14 @@
   -> Network_route6 <| |>
   -> Exec['apply-network-config']
 
-  exec {'apply-network-config':
-    command => 'apply_network_config.sh',
+  if($::operatingsystem == 'poky-stx') {
+    exec {'apply-network-config':
+      command => 'apply_network_config_poky.sh',
+    }
+  } else {
+    exec {'apply-network-config':
+      command => 'apply_network_config.sh',
+    }
   }
 }
 
