diff --git a/ext/openssl/ossl_ssl_session.c b/ext/openssl/ossl_ssl_session.c
index a7437ca..a89488d 100644
--- a/ext/openssl/ossl_ssl_session.c
+++ b/ext/openssl/ossl_ssl_session.c
@@ -75,10 +75,16 @@ static VALUE ossl_ssl_session_initialize(VALUE self, VALUE arg1)
 #if HAVE_SSL_SESSION_CMP == 0
 int SSL_SESSION_cmp(const SSL_SESSION *a,const SSL_SESSION *b)
 {
-    if (a->ssl_version != b->ssl_version ||
-	a->session_id_length != b->session_id_length)
-	return 1;
-    return memcmp(a->session_id,b-> session_id, a->session_id_length);
+	unsigned int a_len;
+	const unsigned char *a_sid = SSL_SESSION_get_id(a, &a_len);
+	unsigned int b_len;
+	const unsigned char *b_sid = SSL_SESSION_get_id(b, &b_len);
+	
+	if (SSL_SESSION_get_protocol_version(a) != SSL_SESSION_get_protocol_version(b))
+		return 1;
+	if (a_len != b_len)
+		return 1; 
+	return CRYPTO_memcmp(a_sid, b_sid, a_len);
 }
 #endif
 
