diff --git a/syslog-ng-config/files/syslog-ng.conf b/syslog-ng-config/files/syslog-ng.conf
index 43a4067..b1bab2c 100644
--- a/syslog-ng-config/files/syslog-ng.conf
+++ b/syslog-ng-config/files/syslog-ng.conf
@@ -183,7 +183,7 @@ destination d_newscrit   { file("/var/log/news/news.crit"); };
 destination d_newserr    { file("/var/log/news/news.err"); };
 destination d_newsnotice { file("/var/log/news/news.notice"); };
 
-# Some `catch-all' logfiles.
+# Some 'catch-all' logfiles.
 #
 destination d_debug    { file("/var/log/debug"); };
 destination d_error    { file("/var/log/error"); };
