From 09c0d3deaed7aa4aa6ec07dd89c4233833133c72 Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Mon, 27 Apr 2020 11:53:02 +0800
Subject: [PATCH 2/4] Set OCF_ROOT_DIR to ${libdir}/ocf

Signed-off-by: Bian Naimeng <biannm@cn.fujitsu.com>

Rebase for 4.1.1

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 113503e..e1ed2e8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -413,7 +413,7 @@ dnl Variables needed for substitution
 AC_CHECK_HEADERS(heartbeat/glue_config.h)
 
 if test "$ac_cv_header_heartbeat_glue_config_h" = "yes"; then
-  OCF_ROOT_DIR=`extract_header_define heartbeat/glue_config.h OCF_ROOT_DIR`
+  OCF_ROOT_DIR="${libdir}/ocf"
 else
   enable_libnet=no
 fi
-- 
2.7.4

