diff -ru a/manifests/extras.pp b/manifests/extras.pp
--- a/manifests/extras.pp	2020-03-05 16:52:24.718267271 +0800
+++ b/manifests/extras.pp	2020-03-05 17:09:43.086295569 +0800
@@ -106,7 +106,8 @@
 
   $packages_run = $::operatingsystem ? {
     /(?i:RedHat|CentOS|Scientific|Amazon|Linux)/ => [ 'yum repolist' , 'rpm -qa' ] ,
-    /(?i:Debian|Ubuntu|Mint|poky)/               => [ 'apt-config dump' , 'apt-cache stats' , 'apt-key list' , 'dpkg -l' ],
+    /(?i:poky)/ => [ 'rpm -qa' ] ,
+    /(?i:Debian|Ubuntu|Mint)/               => [ 'apt-config dump' , 'apt-cache stats' , 'apt-key list' , 'dpkg -l' ],
     /(Solaris)/                                  => [ 'pkginfo' ],
     /(Archlinux)/                                => [ 'pacman -Qet' ],
     default                                      => [ 'echo' ],
