From d42dc102aaca386f314808e771a56b9ecbd29876 Mon Sep 17 00:00:00 2001
From: "Sar Ashki, Babak" <Babak.SarAshki@windriver.com>
Date: Tue, 25 Feb 2020 13:58:15 -0800
Subject: [PATCH] stx-metal: remove argparse requirement from inventory

---
 inventory/inventory/requirements.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/requirements.txt b/requirements.txt
index 993f224..82170c3 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -6,7 +6,6 @@ pbr>=2.0 # Apache-2.0
 SQLAlchemy
 amqplib>=0.6.1
 anyjson>=0.3.3
-argparse
 eventlet==0.20.0
 futurist>=1.2.0 # Apache-2.0
 greenlet>=0.3.2 # MIT
-- 
2.23.0

