diff -ruN a/mtce/src/pmon/pmonHdlr.cpp b/mtce/src/pmon/pmonHdlr.cpp
--- a/mtce/src/pmon/pmonHdlr.cpp	2020-05-15 14:41:42.271215598 +0000
+++ b/mtce/src/pmon/pmonHdlr.cpp	2020-05-18 07:11:16.126154700 +0000
@@ -1881,10 +1881,7 @@
              * we can turn this into a customer alarm/log.
              * Degrade is acheived by not sending the pulses to the watcher.
              */
-            if ( ctrl_ptr->event_mode == true )
-            {
-                pmon_send_pulse ( );
-            }
+            pmon_send_pulse ( );
         }
 
         /* Avoid pmond thrashing trying to recover processes during
@@ -1930,11 +1927,9 @@
             {
                 hostwd_port_init();
             }
-            if ( ctrl_ptr->event_mode == true )
-            {
-                pmon_send_hostwd ( );
-                pmonTimer_hostwd.ring = false;
-            }
+
+	    pmon_send_hostwd ( );
+	    pmonTimer_hostwd.ring = false;
         }
 
         /* Run Get Events by audit timer */
