diff --git a/manifests/params.pp b/manifests/params.pp
index 3254d63..20f91ef 100644
--- a/manifests/params.pp
+++ b/manifests/params.pp
@@ -133,6 +133,7 @@ class postgresql::params inherits postgresql::globals {
         $needs_initdb = pick($needs_initdb, false)
         $service_name = $::operatingsystem ? {
           'Debian' => pick($service_name, 'postgresql'),
+          'poky-stx'   => pick($service_name, 'postgresql'),
           'Ubuntu' => $::lsbmajdistrelease ? {
             /^10/ => pick($service_name, "postgresql-${version}"),
             default => pick($service_name, 'postgresql'),
