diff --git a/lib/puppet/parser/functions/postgresql_acls_to_resources_hash.rb b/lib/puppet/parser/functions/postgresql_acls_to_resources_hash.rb
index 7fa7854..d564efc 100644
--- a/lib/puppet/parser/functions/postgresql_acls_to_resources_hash.rb
+++ b/lib/puppet/parser/functions/postgresql_acls_to_resources_hash.rb
@@ -30,7 +30,7 @@ module Puppet::Parser::Functions
 
     offset = args[2].to_i
     raise(Puppet::ParseError, "#{func_name}: third argument must be a number") \
-      unless offset.instance_of? Fixnum
+      unless offset.instance_of? Integer
 
     resources = {}
     acls.each do |acl|
